src/components/ElectionCandidatesTableSection/ElectionCandidatesTableSection.stories.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+import React from "react";
+import { mockCandidatesList } from "../../util/mocks";
+import { ElectionCandidatesTableSection } from "./ElectionCandidatesTableSection";
+import cssClasses from "./ElectionCandidatesTableSection.module.scss";
+
+export default {
+  title: "Election results candidates tables section",
+  component: ElectionCandidatesTableSection,
+};
+
+export const SimpleExample = () => {
+  return (
+    <ElectionCandidatesTableSection
+      classes={cssClasses}
```

Comment:
You don't need to pass the classes at the component's usage site :) It doesn't hurt anything either.